styles.css

```diff
@@ -199,6 +199,19 @@ body {
   font-size: 1.35rem;
 }
 
+.save-button:hover {
+  cursor: pointer;
+}
+
+.save-button.disabled {
+  background-color: #696969;
+  border: #696969;
```

Comment:
This is pretty clear, and I believe it's working correctly, but it might be best practice to keep this as 'border-color:' since that's all we're using it for.